### PR TITLE
feat: Add example analysis datasets and notebooks

### PR DESCRIPTION
**Pull request by yangchoo**
Adds example absorbance datasets for Absorbance and Kinetics
demonstration.
Augment existing objects notebook showcasing dataset access.
Add new Analysis notebook to show analysis functions.

### CODE REVIEW

**Comment by yangchoo** [comment]
In terms of reviewing, its probably easiest to look at the analysis notebook at https://github.com/strateos/transcriptic/blob/e923e398a9f3611138019bf42e13583a659bc88d/notebooks/analysis.ipynb

File: transcriptic/sampledata/_data/d123-raw-bak.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "a1": [0.038], "a2": [0.034], "a3": [0.059],
+  "b1": [2.775], "b2": [3.041], "b3": [3.111],
+"c1": [2.893], "c2": [3.052], "c3": [2.986]
```
Comment (aota001) [inline transcriptic/sampledata/_data/d123-raw-bak.json:4]: nit: indentation

**Review by aota001 — APPROVED** [review]
just a nit. lgtm

File: transcriptic/sampledata/_data/d123-raw-bak.json (hunk 1 above)
Comment (yangchoo) [reply transcriptic/sampledata/_data/d123-raw-bak.json:4]: oops. thanks!

**Review by aota001 — APPROVED** [review]

**Pull request merged**